src/app/loadout-builder/generated-sets/TierSelect.tsx

```diff
@@ -186,7 +187,7 @@ function MinMaxSelectInner({
         t('LoadoutBuilder.SelectMax')
        */}
       {_.range(min, max + 1).map((tier) => (
-        <option key={tier} value={tier}>
+        <option key={tier} value={tier} className={clsx(ignored && styles.hiddenOption)}>
```

Comment:
Technically you don't need `clsx` for this (though our babel plugin will optimize it out). The clsx way of doing it is:

```js
className={{
 [styles.hiddenOption]: ignored
}}
```

Reply:
I didn't know you could do it like that. I am a sucker for doing it correctly so I will change it now.